### PR TITLE
Tournament updates

### PR DESCRIPTION
**Pull request by Eudaldca**
Using priority 32 like other mini events

### CODE REVIEW

File: models/trophies/attack-on-point-winner.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+name: The Quad Before Christmas Winner
```
Comment (Palmidence) [inline models/trophies/attack-on-point-winner.yml]: Change to `name: Attack on Point Winner`

File: models/trophies/incrediwools-second.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+name: RotG Runner-Up
```
Comment (Palmidence) [inline models/trophies/incrediwools-second.yml]: Change to `name: Incrediwools Runner-Up`

File: models/trophies/incrediwools-winner.yml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+name: RotG Winner
```
Comment (Palmidence) [inline models/trophies/incrediwools-winner.yml]: Change to `name: Incrediwools Winner`

**Comment by Matic0B** [comment]
Since "Incrediwools" isnt as long as "Return of the gamemodes" it probably shouldnt be shortened?

**Comment by Eudaldca** [comment]
Fixed @Matic0B

**Pull request merged**